style.css

```diff
@@ -19,6 +17,12 @@ div {
     "div4 div5 div6"
     "div7 div8 div9";
 }
+
+.div-background {
+  border: 2px solid red;
+  background-color: teal;
```

Comment:
here is a new sub-class for the background and border of the big outer box. Taken out of the general div class since the spinner inherits its value. So that the spinner can have a transparent background